packages/atomic/src/components/insight/atomic-insight-refine-modal/atomic-insight-refine-modal.tsx

```diff
@@ -0,0 +1,177 @@
+import {Component, h, State, Prop, Element, Watch, Host} from '@stencil/core';
```

Comment:
There's a lot of code duplication with the atomic-refine-modal that we could be able to abstract

Reply:
(In a separate PR as this one is quite big already 😄 )